ee/desktop/notify/notify_windows.go

```diff
@@ -4,19 +4,62 @@
 package notify
 
 import (
+	"github.com/go-kit/kit/log"
 	"gopkg.in/toast.v1"
 )
 
-func (d *DesktopNotifier) SendNotification(title, body string) error {
+type windowsNotifier struct {
+	iconFilepath string
+	logger       log.Logger
+	interrupt    chan struct{}
+}
+
+func newOsSpecificNotifier(logger log.Logger, iconFilepath string) *windowsNotifier {
+	return &windowsNotifier{
+		iconFilepath: iconFilepath,
+		logger:       logger,
+		interrupt:    make(chan struct{}),
+	}
+}
+
+// Listen doesn't do anything on Windows -- the `launch` variable in the notification XML
+// automatically handles opening URLs for us.
+func (w *windowsNotifier) Listen() error {
+	for {
+		select {
+		case <-w.interrupt:
+			return nil
+		}
+	}
```

Comment:
I don't think we need the for loop since it's going to block either way?

```suggestion
    <-w.interrupt
    return nil
```